src/audio/play.rs

```diff
@@ -29,46 +30,39 @@ pub fn try_play_once(source: &Source, endpoint: &Output) -> Result<(), DecoderEr
     }
 }
 
-/// Play a sound once.
+/// Play a sound once.  A volume of 1.0 is unchanged, while 0.0 is silent.
 ///
 /// This may silently fail, in order to get error information use `try_play_once`.
-pub fn play_once(source: &Source, endpoint: &Output) {
+pub fn play_once(source: &Source, volume: f32, endpoint: &Output) {
     // This absurd construct is here to keep warnings about an unused result from appearing.
-    if let Ok(()) = try_play_once(source, endpoint) {}
+    let _ = try_play_once(source, volume, endpoint);
 }
 
-/// Play a sound n times.
+/// Play a sound n times.  A volume of 1.0 is unchanged, while 0.0 is silent.
 ///
 /// This may silently fail, in order to get error information use `try_play_n_times`.
-pub fn play_n_times(source: &Source, endpoint: &Output, n: u16) {
+pub fn play_n_times(source: &Source, volume: f32, endpoint: &Output, n: u16) {
     // This absurd construct is here to keep warnings about an unused result from appearing.
-    if let Ok(()) = try_play_n_times(source, endpoint, n) {}
+    let _ = try_play_n_times(source, volume, endpoint, n);
```

Comment:
Could we still have a printed error message here?

Reply:
Sure!